features/step_definitions/active_docs_steps.rb

```diff
@@ -1,3 +1,40 @@
+# frozen_string_literal: true
+
+Given(/^the provider has (\d) active docs?$/) do |number_active_docs|
+  @active_docs = FactoryGirl.create_list(:api_docs_service, number_active_docs.to_i, account: @provider, service: @provider.default_service)
+end
+
+When /^I try to (create|update) the active docs( of the service)? with (valid|invalid) data$/ do |action, optional_scope, valid|
```

Comment:
the valid/invalid part could be `... (in)?valid data do |..., invalid|` and `invalid` can used as a boolean value.

Reply:
wow this is gold! thank you so much! 💃